docs/en/status-communities/create-a-status-community.md

```diff
@@ -6,9 +6,58 @@ hide:
+
+As a community owner, you can [restrict community access using tokens][configure-your-community-permissions], control which users are added to the community and organize discussion groups in [channels][create-a-channel]. Owners have complete control over their community and are responsible for maintaining its rules.
+
+## Create a community
+
+=== "Desktop"
+
+    1. From the navigation sidebar, click :desktop-communities: **Communities**.
+    1. On top of the content area, click **Create community** and then, click **Create new**.
+    1. Add a **name** and a **short description** of your community.
+    1. Optionally, add a **logo**, **banner**, **colour** palette and **tags**. You can [customize these settings](#customize-your-community) at any time.
+    1. Configure [additional community settings](#configure-additional-settings).
+    1. Click **Next** to complete the [information about your community](#describe-your-community-and-its-rules).
+    1. Click **Create community**.
+
+## Configure additional settings
+
+You can configure these community settings:
+
+| Setting | Description |
+|:---|:---|
+| **Community history service** | Status Communities store up to 30 days of chat history by default. When this option is checked, community members can see messages older than 30 days.</br></br>For more information, check out [About the Community history service][about-the-community-history-service].|
+| **Request to join required** | Community owners must manually approve each join request if this setting is checked. This option applies to [open and private][about-status-communities] communities. |
+| **Any member can pin a message** | Any community member can [pin messages][pin-a-message] when this setting is checked. If the setting is unchecked, only the community owner can pin messages. |
+
+!!! note
+    The Community history service uses the [BitTorrent peer-to-peer network :octicons-tab-external-16:][bittorrent-peer-to-peer-network]{:target="_blank"} to store and retrieve community messages older than 30 days. Status uses a decentralized network and does not store this information.
+
+## Describe your community and its rules
+
+=== "Desktop"
+
+    1. In the **Community introduction and rules**, introduce your community to new members and describe the community guidelines.
+    1. In the **Leaving community message**, write the message sent to members leaving your community.
+    1. Click **Create community** if you haven't already.
+
+## Customize your community
+
+After you create a community, you can customize its settings at any time.
+
+=== "Desktop"
+
+    1. From the navigation sidebar, click your community.
+    1. On top of the channel sidebar, click your community logo.
+    1. Click :desktop-overview: **Overview**.
```

Comment:
```suggestion
    1. Click :desktop-overview: **Overview**.
    1. Expand the window and click **Edit Community**.
```
On the current Status desktop there's this additional step. And this button is hidden if the window is not big enough.

Reply:
Thank you, @cheny0 -- I'm adding your suggestion, but I think the MVP will fix these problems with buttons and other UI controls showing up beyond the app window limits.